### DISCUSSION TITLE
Local setup without KeyCloak

### DISCUSSION THREAD

**Discussion by IvanDrag0** [NUWCDIVNPT/stig-manager#1084] [General]
If I'd like to setup a local instance of stig-manager for testing/evaluation, is it possible to do so with local authentication (or no authentication at all) instead of having to use KeyCloak?

**Comment by cd-rite** [comment]
HI @IvanDrag0 STIGMan will not operate without an external identity provider.
One option that may help is our very simple Quick Start Orchestration demo offered on Docker Hub, but this should not be used for production purposes:
https://hub.docker.com/repository/docker/nuwcdivnpt/stig-manager/general
That orchestration uses our demo keycloak container with a few pre-built users that have all their passwords set to "password":
https://hub.docker.com/repository/docker/nuwcdivnpt/stig-manager-auth/general
